examples/robust_training.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""Robust Training in JAXOpt.
+
+The following code trains a convolutional neural network (CNN) to be robust
+with respect to the fast sign gradient (FSG) method, which is an attack
+on the L-infinity ball.
+
+
```

Comment:
If that's possible to do in just a few lines, I would explain the gist of the idea here.